### ISSUE TITLE
Not working in Google Cloud Run

### ISSUE THREAD

**Issue by KimMachineGun** [KimMachineGun/automemlimit#7]
Hey, I'd also be interested in this. Well, not necessarily sigar, but rather having automemlimit "just work" even outside a container.

For my use case, I am using Google Cloud run, and currently automemlimit doesn't work because it can't see it is running in a container. Specifically, I get the message

```
2023/04/23 01:40:07 failed to set GOMEMLIMIT: open /sys/fs/cgroup/cgroup.controllers: no such file or directory.
```

_Originally posted by @mbyio in https://github.com/KimMachineGun/automemlimit/issues/4#issuecomment-1518915716_

**Comment by KimMachineGun** [comment]
@mbyio
Thanks again for sharing this issue.

This issue has been fixed in #8 (not yet released).
It will be released as v0.2.6 after a few days of testing, and it would be great if you could double-check the changes in the main branch.

You can use the fixed version right away with the command below.
`go get github.com/KimMachineGun@da53042`

**Comment by mbyio** [comment]
Wow, that was fast haha. Yes, it looks like that fixed it. Thanks!
